README.md

```diff
@@ -54,9 +54,13 @@ Holdout Set
 
 RMSE = root mean squared error (what you try to minimize) its in the same units as what you are trying to predict.
 
-MAE
+MAE - evaluation metric used for regression models, 
```

Comment:
L1 loss function

Reply:
MAE - evaluation metric used for regression models, a method for evaluating accuracy, it is a loss function calculated by summing the difference between predicted and actual values (Sigma(y_pred - y)), and is referred to as the L1 loss function. First seen in the KNN lecture.